code/core/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "resolveJsonModule": true,
     "moduleResolution": "bundler",
-    "module": "ES2022",
+    "module": "esnext",
```

Comment:
this was necessary to support import attributes like `await import('storybook/package.json', { with: { type: 'json' }, });` and is recommended when you have a bundler that supports all syntax but compiles it down, which I guess we do